DotnetFoundation/DotnetFoundation.Application/Models/Common/PagingRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DotnetFoundation.Application.Models.Common;
+public class PagingRequest
+{
+    const int MaxPageSize = 50;
```

Comment:
Is this a public property? Access spec is not given here.

Reply:
Changed this to private